internal/tui/editor.go

```diff
@@ -0,0 +1,68 @@
+package tui
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/google/shlex"
+)
+
+// I'm sure these 10 lines are controversial to some. If you find yourself
+// caring about this, just set $EDITOR!
+var editors = []string{
+	// graphical editors
+	"code",
+	"subl",
+	"gedit",
+	"nodepad++",
+
+	// editors that mere mortals might not remember how to exit
+	//
+	// also, editors that might not take kindly to being run from within a
+	// terminal in *another* editor
+	"vim",
+	"vi",
+	"emacs",
+	"helix",
+
+	// everyone has these, right?
+	"nano",
+	"pico",
```

Comment:
🤣

Reply:
I didn't know Pico existed. 😆
I'm so used to vscode I feel like half a dev every time the magic of nvim being used is demoed. 😆

I did have an ergodox with blank keycaps to work on my geek cred but decided it was too much for my feeble brain.